### PR TITLE
[stable/airflow] Add DAG git ref to the scheduler and web pod annotat…

### PR DESCRIPTION
**Pull request by omonnier**
…ions

This will trigger a pod restart whenever the git ref
changes on the git project containing the DAGS.

Signed-off-by: Olivier Monnier <5500018+omonnier@users.noreply.github.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/airflow/templates/deployments-scheduler.yaml [hunk 1]
```diff
@@ -37,6 +37,9 @@ spec:
         checksum/config-scripts: {{ include (print $.Template.BasePath "/configmap-scripts.yaml") . | sha256sum }}
         checksum/config-variables-pools: {{ include (print $.Template.BasePath "/configmap-variables-pools.yaml") . | sha256sum }}
         checksum/secret-connections: {{ include (print $.Template.BasePath "/secret-connections.yaml") . | sha256sum }}
+        {{- if .Values.dags.git.ref }}
```
Comment (gsemet) [inline stable/airflow/templates/deployments-scheduler.yaml]: Please conditionate with 
```
{{- if and(.Values.dags.git.url, .Values.dags.git.ref) }}
```

Comment (omonnier) [reply stable/airflow/templates/deployments-scheduler.yaml]: done

**Pull request merged**